ui/src/views/Rules/ListRules/index.jsx

```diff
@@ -83,10 +84,13 @@ const useStyles = makeStyles(theme => ({
     left: 0,
     padding: `${theme.spacing(4)}px ${theme.spacing(4)}px 0`,
     display: 'flex',
-    justifyContent: 'flex-end',
+    justifyContent: 'center',
```

Comment:
praise: thank you for restoring this - I think it was changed unintentionally in a recent PR.